modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -27,7 +27,7 @@ import (
 // Keeper defines the IBC interchain accounts controller keeper
 type Keeper struct {
 	storeKey       storetypes.StoreKey
-	cdc            codec.BinaryCodec
+	cdc            codec.Codec
```

Comment:
not strictly necessary since the controller doesn't do JSON marshaling, what prompted this change?

Reply:
It was your comment that prompted the change: https://github.com/cosmos/ibc-go/pull/3796#discussion_r1244979441

Lmk if you want me to revert the change before I merge today